src/lib/utils.js

```diff
@@ -22,8 +25,31 @@ class Utils {
 
   }
 
-}
+  static async readJson(path) {
+    try {
+      return JSON.parse(
+        await fs.readFile(path, {encoding: 'utf8'})
+      );
+    } catch (error){
+        console.error(error);
+    }
+  }
+
+  static async getContext(){
+    return await Utils.readJson(path.resolve(path.resolve(), './src/lib/oa.jsonld'));
```

Comment:
Isn't there a website we can load this from? But we can put this in for now.

Reply:
it seemed more efficient to cache it locally. It's also better practice as every network request is an attack vector :)

Reply:
We can load once at startup then cache that (like we did for org tree last time), but yeah, it's fine for now